packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -721,8 +721,9 @@ class FlutterPlugin implements Plugin<Project> {
                 return
             }
             // Flutter module included as a subproject in add to app.
-            Project appProject = project.rootProject.findProject(':app')
-            assert appProject != null
+            String hostAppProjectName = project.rootProject.hasProperty('flutter_host_app_project_name') ? project.rootProject.property('flutter_host_app_project_name')  : "app"
```

Comment:
sorry for not mentioning this before, but most Gradle properties have a namespace. What about if this becomes `flutter.host_app_project_name`, so that it follows the convention?

Reply:
can you share the example of any Gradle properties in this namespace? 

Do you mean simple rename to 
`project.hasProperty('flutter.host_app_project_name') `
or something like
`project.flutter.host_app_project_name` ?

Reply:
A simple rename should be fine.

```
android.useAndroidX=true
android.enableJetifier=true
// Flutter flag
flutter.hostAppProjectName=SampleApp
```

Reply:
got it 👍